src/main/java/com/aws/greengrass/deployment/DeploymentDirectoryManager.java

```diff
@@ -41,6 +41,7 @@ public class DeploymentDirectoryManager {
 
     private static final String PREVIOUS_SUCCESS_LINK = "previous-success";
     private static final String PREVIOUS_FAILURE_LINK = "previous-failure";
+    private static final String PREVIOUS_REJECTED_LINK = "previous-rejected";
```

Comment:
I don't think we really want a directory for rejected deployments

Reply:
I can remove this as well. I thought we are maintaining records of both success and failure, then we might also be interested in storing rejected too.